R-package/tests/testthat/test_dmatrix.R

```diff
@@ -103,8 +103,7 @@ test_that("xgb.DMatrix: saving, loading", {
   on.exit(unlink(tmp_file))
   expect_true(xgb.DMatrix.save(dtest1, tmp_file))
   # read from a local file
-  expect_output(dtest3 <- xgb.DMatrix(tmp_file), "entries loaded from")
-  expect_output(dtest3 <- xgb.DMatrix(tmp_file, silent = TRUE), NA)
+  expect_output(dtest3 <- xgb.DMatrix(tmp_file), NA)
```

Comment:
I think there's no need to remove these tests. Perhaps a new one could be added checking that there is no log when the setting is changed globally and arg. `silent` is not passed.

Reply:
Brought back the tests.